libraries/chain/thread_utils.cpp

```diff
@@ -19,8 +20,15 @@ void named_thread_pool::start( size_t num_threads, on_except_t on_except ) {
    _ioc_work.emplace( boost::asio::make_work_guard( _ioc ) );
    _ioc.restart();
    _thread_pool.reserve( num_threads );
+
+   // capture name since name_prefix is invalid in debugger, use abieos/tools/num2name to get name
+   std::string sanitized_name = fc::to_lower(_name_prefix);
+   sanitized_name = sanitized_name.substr(0, 12);
+   name desc(string_to_uint64_t(sanitized_name));
+
    for( size_t i = 0; i < num_threads; ++i ) {
-      _thread_pool.emplace_back( [&ioc = _ioc, &name_prefix = _name_prefix, on_except, i]() {
+      _thread_pool.emplace_back( [desc, &ioc = _ioc, &name_prefix = _name_prefix, on_except, i]() {
+         (void)desc;
```

Comment:
It would be add a comment here that `desc` is just for putting on the stack for debug.

Reply:
Added comment

Reply:
Thanks.